app/helpers/editor_helper.rb

```diff
@@ -0,0 +1,7 @@
+module EditorHelper
+  def template_name_attribute(template)
+    return 'provisioning_template[template]' if template == 'ProvisioningTemplate'
+    return 'report_template[template]' if template == 'ReportTemplate'
+    return 'ptable[template]' if template == 'Ptable'
```

Comment:
how do we handle templates introduced by plugins (e.g. JobTemplate form REX plugin), I suppose this is used for form element name, could perhaps this work?
```
"#{template.class.underscore}[template]"
```

though I think "template" attribute will not always the name of attribute for which we mount the editor